### PR TITLE
feat: Add virtual network rules for sql servers

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

File: resources/services/network/public_ip_addresses.go [hunk 1]
```diff
@@ -27,14 +27,14 @@ func NetworkPublicIPAddresses() *schema.Table {
 				Resolver:    client.ResolveAzureSubscription,
 			},
 			{
-				Name:          "extended_location_name.",
+				Name:          "extended_location_name",
```
Comment (roneli) [inline resources/services/network/public_ip_addresses.go:30]: wouldn't this require a rename on the column?

**Review by roneli — CHANGES_REQUESTED** [review]
Found something that you fixed in your PR, which seems like a bug inserted in another PR, verify me on how the column name is created, and if it does have "." create a rename

File: resources/services/network/public_ip_addresses.go (hunk 1 above)
Comment (disq) [reply resources/services/network/public_ip_addresses.go:30]: https://github.com/cloudquery/cq-provider-azure/pull/114 introduced this bug, docs and 2 column names have `.` added to them, but no migrations etc.

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**